specs/disciplina/educator/api/student.yaml

```diff
@@ -0,0 +1,414 @@
+      produces:
+        - application/json
+      responses:
+        200:
+          description: successful operation
+          schema:
+            $ref: '#/definitions/Assignment'
+        404:
+          description: Assignment with given wasn not found (or a user has no rights to look it up)
+
+  /submissions:
+    get:
+      tags:
+        - submissions
+      summary: Get student\'s submissions
+      description: Gets a list of student\'s submissions. Filter parameters are used to specify
+        specific course, assignment, etc.
+      operationId: getSubmissions
+      parameters: 
+        - $ref: '#/parameters/CourseId'
+        - $ref: '#/parameters/AssignmentHash'
+        - $ref: '#/parameters/DocumentType'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Submission'
+    post:
+      tags:
+        - submissions
+      summary: Make a new submission
+      description: Posts a new submission with a given body. Request body should contain valid student\'s signature of submission contents, otherwise an error will be raised.
+      operationId: makeSubmission
+      produces:
+        - application/json
+      consumes:
+        - application/json
+      parameters:
+        - in: body
+          name: submission
+          description: A submission
+          required: true
+          schema:
+            $ref: '#/definitions/Submission'
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/Submission'
+        '403':
+          description: Either submission body or submission signature is invalid
+          
+  /submissions/{submissionHash}:
+    get:
+      tags:
+        - submissions
+      summary: Get info about a submission
+      description: Gets a submission data by given submission hash. Returns a 404 if a student tries to get a submission which is not their own.
+      operationId: getSubmission
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/SubmissionHashPath'
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/Submission'
+        '404':
+          description: Submission with given wasn not found (or a user has no rights to look it up)
+    delete:
+      tags:
+        - submissions
+      summary: Delete a submission
+      description: Deletes a submission from a database. Only ungraded submissions can be deleted.
+      operationId: deleteSubmission
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/SubmissionHashPath'
+      responses:
+        '200':
+          description: successful operation
+        '403':
+          description: Cannot delete a graded submission
```

Comment:
Maybe 403 should return the string with the error code so that we can process all 403 responses in a similar way?